### ISSUE TITLE
mysqlWorkbench failing to build

### ISSUE THREAD

**Issue by jasonMoggridge** [NixOS/nixpkgs#6579]
The build for mysqlWorkbench stops part way through with this error:

    make[3]: *** [recordset_text_storage.lo] Error 1
    make[3]: Leaving directory `/tmp/nix-build-mysql-workbench-5.2.47.drv-0/mysql-workbench-gpl-5.2.47-src/backend/wbpublic'
    make[2]: *** [all-recursive] Error 1
    make[2]: Leaving directory `/tmp/nix-build-mysql-workbench-5.2.47.drv-0/mysql-workbench-gpl-5.2.47-src/backend'
    make[1]: *** [all-recursive] Error 1
    make[1]: Leaving directory `/tmp/nix-build-mysql-workbench-5.2.47.drv-0/mysql-workbench-gpl-5.2.47-src'
    make: *** [all] Error 2
    builder for ‘/nix/store/3wn18bn045f038hy0h7pvri3hk9085yw-mysql-workbench-5.2.47.drv’ failed with exit code 2
    cannot build derivation ‘/nix/store/gh0x65k0y046hflfck3xw6amz03g8fym-system-path.drv’: 1 dependencies couldn't be built
    cannot build derivation ‘/nix/store/qvj2fi7k4b4szd3j2mcd0bkjrx72zai7-nixos-15.05pre58123.9775f46.drv’: 1 dependencies couldn't be built
    error: build of ‘/nix/store/qvj2fi7k4b4szd3j2mcd0bkjrx72zai7-nixos-15.05pre58123.9775f46.drv’ failed

**Comment by quamilek** [comment]
I have a similar problem
```
sqlide/recordset_text_storage.cpp:352:67:   required from here
./sqlide/sqlide_generics.h:235:15: warning: unused parameter 'x' [-Wunused-parameter]
make[3]: *** [recordset_text_storage.lo] Error 1
make[3]: Leaving directory `/tmp/nix-build-mysql-workbench-5.2.47.drv-0/mysql-workbench-gpl-5.2.47-src/backend/wbpublic'
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory `/tmp/nix-build-mysql-workbench-5.2.47.drv-0/mysql-workbench-gpl-5.2.47-src/backend'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/tmp/nix-build-mysql-workbench-5.2.47.drv-0/mysql-workbench-gpl-5.2.47-src'
make: *** [all] Error 2
builder for ‘/nix/store/cfa6h117dc595r49l4wfp81zp5msfnky-mysql-workbench-5.2.47.drv’ failed with exit code 2
error: build of ‘/nix/store/cfa6h117dc595r49l4wfp81zp5msfnky-mysql-workbench-5.2.47.drv’ failed
```

**Comment by lethalman** [comment]
The above logs are useless, they don't show the actual error. Please look for lines with `error:`. I'm not able to reproduce this problem with git master. What's your nixpkgs commit and on which arch are you building it?

**Comment by grahamc** [comment]
I think this can be closed, as it installs just fine on 15.09.

**Comment by fpletz** [comment]
Also works for me on master. Closing.
